.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "submodules/quickstart-atlassian-services"]
 	path = submodules/quickstart-atlassian-services
-	url = https://github.com/aws-quickstart/quickstart-atlassian-services.git
```

Comment:
This may be accidental? We should be using HTTPS in submodule URLs

Reply:
Oh dear I'm not sure how this happened 🤦‍♂

Reply:
Done